draganddrop/draganddrop/MainPage.xaml.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
```

Comment:
There are a number of samples that don’t use the template.  What do you mean they will be destroyed next time there is a template update?

From: Raymond Chen [mailto:notifications@github.com]
Sent: Monday, June 15, 2015 9:09 AM
To: Microsoft/Windows-universal-samples
Subject: Re: [Windows-universal-samples] drag and drop sample (#25)


In draganddrop/draganddrop/MainPage.xaml.cs<https://github.com/Microsoft/Windows-universal-samples/pull/25#discussion_r32436935>:

> @@ -0,0 +1,62 @@

> +﻿using System;

This sample does not follow the standard sample template. It will be destroyed the next time there is a template update.

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/Windows-universal-samples/pull/25/files#r32436935>.